tokio-stream/src/stream_ext.rs

```diff
@@ -113,6 +113,12 @@ pub trait StreamExt: Stream {
     /// pinning it to the stack using the `pin_mut!` macro from the `pin_utils`
     /// crate.
     ///
+    /// # Cancel safety
```

Comment:
i was going to suggest we capitalize both words, since it's a title...but we don't actually do that anywhere else in the docs. so, we may want to fix that, separately; let's continue with this capitalization for now for consistency.

Reply:
Yah, I copied this from elsewhere in the same crate! Looks like we just need someone with write access to merge!